index.js

```diff
@@ -6,13 +6,15 @@ var balanced = require("balanced-match")
 var colorFn = require("css-color-function")
 var helpers = require("postcss-message-helpers")
 
+var colorFnRegExp = /(^|[^\w\-])color\(/;
+
 /**
  * PostCSS plugin to transform color()
  */
 module.exports = postcss.plugin("postcss-color-function", function() {
   return function(style) {
     style.eachDecl(function transformDecl(decl) {
-      if (!decl.value || decl.value.indexOf("color(") === -1) {
+      if (!decl.value) {
```

Comment:
> If !decl.value will be failed, second test will be checked in transformColor anyway.

That why a simple & stupid but fast test is prefered before invoking anything more complicated.